src/test/java/TestUtils.java

```diff
@@ -0,0 +1,26 @@
+import com.itextpdf.text.DocumentException;
+import com.itextpdf.text.pdf.*;
+import org.apache.commons.io.IOUtils;
+
+import java.io.*;
+import java.nio.charset.Charset;
+import java.util.ArrayList;
+
+public class TestUtils {
+
+    public static boolean checkDocumentIsFlat(InputStream fileFlattened) {
+        boolean isflat;
+        try {
+            PdfReader reader = new PdfReader(fileFlattened);
+            ByteArrayOutputStream bos = new ByteArrayOutputStream();
+            PdfStamper stamper = new PdfStamper(reader, bos);
+            AcroFields fields = stamper.getAcroFields();
+            isflat = fields.setField("name", "ChangedName");
+            return isflat;
+        } catch (IOException | DocumentException e) {
```

Comment:
test utils don't have to catch exceptions. Just have them propagated to the test method and JUnit will fail the test when exception ocurrs.

Reply:
Good point :). Fixed